arm-storageimportexport/2016-11-01/swagger/storageimportexport.json

```diff
@@ -0,0 +1,1194 @@
+          "items": {
+            "type": "string"
+          }
+        },
+        "alternateLocations": {
+          "type": "array",
+          "description": "A list of location IDs that should be used to ship shipping drives to for jobs created against the current location. If the current location is active, it will be part of the list. If it is temporarily closed due to maintenance, this list may contain other locations.",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Location": {
+      "description": "Provides information about an Azure data center location.",
+      "properties": {
+        "id": {
+          "description": "Specifies the resource identifier of the location.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Specifies the name of the location. Use List Locations to get all supported locations.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Specifies the resource type of the location.",
+          "type": "string"
+        },
+        "properties": {
+          "description": "Describes the properties of a Location.",
+          "$ref": "#/definitions/LocationProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ReturnAddress": {
+      "description": "Specifies the return address information for the job.",
+      "required": [
+        "recipientName",
+        "streetAddress1",
+        "city",
+        "postalCode",
+        "countryOrRegion",
+        "phone",
+        "email"
+      ],
+      "properties": {
+        "recipientName": {
+          "type": "string",
+          "description": "The name of the recipient who will receive the hard drives when they are returned."
+        },
+        "streetAddress1": {
+          "type": "string",
+          "description": "The first line of the street address to use when returning the drives."
+        },
+        "streetAddress2": {
+          "type": "string",
+          "description": "The second line of the street address to use when returning the drives."
+        },
+        "city": {
+          "type": "string",
+          "description": "The city name to use when returning the drives."
+        },
+        "stateOrProvince": {
+          "type": "string",
+          "description": "The state or province to use when returning the drives."
+        },
+        "postalCode": {
+          "type": "string",
+          "description": "The postal code to use when returning the drives."
+        },
+        "countryOrRegion": {
+          "type": "string",
+          "description": "The country or region to use when returning the drives."
+        },
+        "phone": {
+          "type": "string",
+          "description": "Phone number of the recipient of the returned drives. Is there any requirement for the country code? Is there any specific format that the server expects?"
```

Comment:
I think this description includes a TODO from Amar's review (just missing the TODO):

 Is there any requirement for the country code? Is there any specific format that the server expects?